identifiers/country-hk/functional_constituencies.csv

```diff
@@ -1,30 +1,31 @@
-ocd-division/country:hk/functional_constituency:textiles_and_garment,Textiles and Garment
-ocd-division/country:hk/functional_constituency:wholesale_and_retail,Wholesale and Retail
-ocd-division/country:hk/functional_constituency:information_technology,Information Technology
-ocd-division/country:hk/functional_constituency:catering,Catering
-ocd-division/country:hk/functional_constituency:district_council_first,District Council (First)
-ocd-division/country:hk/functional_constituency:district_council_second,District Council (Second)
+id,name,validThrough,validFrom
+ocd-division/country:hk/functional_constituency:heung_yee_kuk,Heung Yee Kuk,,
+ocd-division/country:hk/functional_constituency:agriculture_and_fisheries,Agriculture and Fisheries,,
+ocd-division/country:hk/functional_constituency:insurance,Insurance,,
+ocd-division/country:hk/functional_constituency:transport,Transport,,
+ocd-division/country:hk/functional_constituency:education,Education,,
+ocd-division/country:hk/functional_constituency:legal,Legal,,
+ocd-division/country:hk/functional_constituency:accountancy,Accountancy,,
+ocd-division/country:hk/functional_constituency:medical,Medical,,
+ocd-division/country:hk/functional_constituency:health_services,Health Services,,
+ocd-division/country:hk/functional_constituency:engineering,Engineering,,
+ocd-division/country:hk/functional_constituency:architectural_surveying_and_planning,"Architectural, Surveying, Planning and Landscape",,
+ocd-division/country:hk/functional_constituency:labour,Labour,,
+ocd-division/country:hk/functional_constituency:social_welfare,Social Welfare,,
+ocd-division/country:hk/functional_constituency:real_estate_and_construction,Real Estate and Construction,,
+ocd-division/country:hk/functional_constituency:tourism,Tourism,,
+ocd-division/country:hk/functional_constituency:commercial_first,Commercial (First),,
+ocd-division/country:hk/functional_constituency:commercial_second,Commercial (Second),,
+ocd-division/country:hk/functional_constituency:industrial_first,Industrial (First),,
+ocd-division/country:hk/functional_constituency:industrial_second,Industrial (Second),,
+ocd-division/country:hk/functional_constituency:finance,Finance,,
+ocd-division/country:hk/functional_constituency:financial_services,Financial Services,,
+ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication",,
+ocd-division/country:hk/functional_constituency:import_and_export,Import and Export,,
+ocd-division/country:hk/functional_constituency:textiles_and_garment,Textiles and Garment,,
+ocd-division/country:hk/functional_constituency:wholesale_and_retail,Wholesale and Retail,,
+ocd-division/country:hk/functional_constituency:information_technology,Information Technology,2021,
+ocd-division/country:hk/functional_constituency:technology_and_innovation,Technology and Innovation,,2021

```

Comment:
I believe the timeline for the abolishing of the old constituency and for the forming of the new constituency would be the same as for the geographical constituencies (See PR #272).

Please update these to mach the `2021-12-19` date

Reply:
Hello, thank you for the response, I've updated the date from 2021 to 2021-12-19